block-storage-attacher/tests/e2e/e2e-tests/block_volume_attach.go

```diff
@@ -144,6 +155,69 @@ var _ = framework.KubeDescribe("[Feature:Block_Volume_Attach_E2E]", func() {
 				logResult("BlockVolumeAttacher-Volume-Test: Attach: PASS\n")
 			}
 
+			/* Port Worx PVC creation */
+			portworxpvcpath = gopath + "/" + portworxpvcpath
+			portworxscpath = gopath + "/" + portworxscpath
+			filestatus, err = fileExists(portworxpvcpath)
+			if err != nil {
+				cleanUP(pvname, pv)
+				logResult("BlockVolumeAttacher-Volume-Test: PVC Creaiton: FAIL\n")
+			} else {
+				logResult("BlockVolumeAttacher-Volume-Test: PVC Creaiton: PASS\n")
+			}
+			Expect(err).NotTo(HaveOccurred())
+
+			/* PVC Creation */
+
+			By("PVC  Creation")
+			if filestatus == true {
+				filepatharg := fmt.Sprintf("%s", portworxpvcpath)
+				filepatharg2 := fmt.Sprintf("%s", portworxscpath)
+				cmd := exec.Command(pvscriptpath, filepatharg, "portworxpvcreate", filepatharg2)
+				var stdout, stderr bytes.Buffer
+				cmd.Stdout = &stdout
+				cmd.Stderr = &stderr
+				err = cmd.Run()
+				if err != nil {
+					cleanUP(pvname, pv)
+					logResult("BlockVolumeAttacher-Volume-Test: Portworx Installtion: FAIL\n")
+				} else {
+					logResult("BlockVolumeAttacher-Volume-Test: Portworx installtion: PASS\n")
+				}
+				Expect(err).NotTo(HaveOccurred())
+			}
+			framework.KubeDescribe("Test mount point: Create mount point, read, write...", func() {
+				//It("PRESTAGE: Test mount point: Create mount point, read, write... [Serial]", func() {
+				By("Creating a claim with a dynamic provisioning annotation")
+				claim := commontest.NewClaim(ns, "portworx-sc", "2Gi")
+				defer func() {
+					c.Core().PersistentVolumeClaims(ns).Delete(claim.Name, nil)
+				}()
+				claim, err := c.Core().PersistentVolumeClaims(ns).Create(claim)
```

Comment:
The PVC will take some time to create like around 2mins if its success case. You may need to keep checking status and then timeout before moving ahead else this may fail every time even if its successful.

Reply:
The call returns only after fail/complete success because it will return the response body with PVC details